pcapng/strictness.py

```diff
@@ -0,0 +1,34 @@
+"""
+Module for alerting the user when attempting to do things with pcapng that
+aren't strictly valid.
+"""
+
+import warnings
+
+from pcapng.exceptions import PcapngStrictnessError, PcapngStrictnessWarning
+
+STRICTNESS_NONE = 0  # No warnings, do what you want
```

Comment:
Make this use enum, please: https://docs.python.org/3/library/enum.html

This is >=3.4 and we support >=3.5, so it should be fine.